src/Root.tsx

```diff
@@ -0,0 +1,20 @@
+import { Route, HashRouter as Router, Routes } from 'react-router-dom';
+import { App } from './App';
+import { HomePage } from './pages/HomePage';
+import { NotFoundPage } from './pages/NotFoundPage';
+import { PeoplePage } from './pages/PeoplePage';
+
+export const Root = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path=":home" element={<HomePage />} />
```

Comment:
There is a potential redundancy here. Both the index route and the `:home` route point to `HomePage`. If `:home` is intended to be a dynamic segment, consider clarifying its purpose or removing it if unnecessary.